tasks/bin/run_external_test

```diff
@@ -11,12 +11,16 @@ include Chef::Mixin::ShellOut
 github_repo = ARGV.shift
 git_thing = ARGV.shift
 
+chef_dir = File.expand_path("../..", File.dirname(__FILE__))
+
 Dir.mktmpdir("chef-external-test") do |dir|
   git_url = "https://github.com/#{github_repo}"
   Dir.rmdir dir
   shell_out!("git clone #{git_url} #{dir}", live_stream: STDOUT)
   Dir.chdir(dir) do
     shell_out!("git checkout #{git_thing}", live_stream: STDOUT)
+    File.open("Gemfile", "a") { |f| f.write("gem 'chef', path: '#{chef_dir}'") }
+    File.open("Gemfile", "a") { |f| f.write("gem 'ohai', git: 'https://github.com/chef/chef.git', branch: 'master'") }
```

Comment:
think you want `ohai.git` here :)

Reply:
yeah, that might work better...  got distracted by immutablearrays...